services/director-v2/src/simcore_service_director_v2/models/comp_runs.py

```diff
@@ -140,10 +144,10 @@ def convert_null_to_empty_metadata(cls, v):
                     "cluster_id": 123,
                     "iteration": 12,
                     "result": "SUCCESS",
-                    "created": "2021-03-01 13:07:34.19161",
-                    "modified": "2021-03-01 13:07:34.19161",
-                    "started": "2021-03-01 8:07:34.19161",
-                    "ended": "2021-03-01 13:07:34.10",
+                    "created": "2021-03-01T13:07:34.191610",
```

Comment:
Q: do these columns are defined in the database as `sa.DateTime(timezone=True)`  i.e. with timezone?
SEE https://github.com/ITISFoundation/osparc-simcore/issues/3797

Reply:
no. and the problem there was I copy/pasted from adminer the date time. and this is not a proper date time format.